src/components/SingleArticle.jsx

```diff
@@ -1,30 +1,53 @@
 import { useParams } from 'react-router-dom';
 import { getArticleById } from '../api';
 import { useState, useEffect } from 'react'
+import { getCommentsByArticleId } from '../api';
+
 
 function SingleArticle (){
     let { article_id } = useParams();
     const [article, setArticle] = useState({})
+    const [comments, setComments] = useState([])
     const [isLoading, setIsLoading] = useState(true)
 
     useEffect(() => {
         getArticleById(article_id).then((article) => {
+            console.log(article)
             setArticle(article)
             setIsLoading(false)
+        })        
+        getCommentsByArticleId(article_id).then((comments) => {
+            setComments(comments)
+            setIsLoading(false)
```

Comment:
two api calls in one useEffect will work, but it would be better to abstract all the comments related code into a CommentList component and fetch the data in that components to avoid future confilcts